sql/1dbFamily.sql

```diff
@@ -1,9 +1,16 @@
+DROP TABLE IF EXISTS `dbFamily_languages`;
+DROP TABLE IF EXISTS `dbFamily_Assistance`;
+DROP TABLE IF EXISTS `dbLanguages`;
+DROP TABLE IF EXISTS `dbAssistance`;
+DROP TABLE IF EXISTS `dbFamily`;
+
 CREATE TABLE `dbFamily` (
     `id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
     `firstName` varchar(256) NOT NULL,
     `lastName` varchar(256) NOT NULL,
     `birthdate` varchar(256) NOT NULL,
     `address` varchar(256) NOT NULL,
+    `neighborhood` varchar(256) NOT NULL,
```

Comment:
If we're adding new columns to dbFamily it would prob be better to do it in a new sql file

Reply:
Create a new file and adding new columns through an alter table statement or do you mean something else?